src/Product of Array Except Self.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var productExceptSelf = function(nums) {
+  let multi = 1;
+  let zeros = 0;
+
+  for (let i = 0; i < nums.length; i++) {
+    const num = nums[i];
+    
+  if (!num) {
+    multi *= 1;
+    zeros++;
+    
+    if (zeros > 1) {
+      multi *= 0;
+      
+      return new Array(nums.length).fill(0);
+    }
+  } else {
+    multi *= num;
+  }
```

Comment:
```suggestion
    if (!num) {
      multi *= 1;
      zeros++;

      if (zeros > 1) return new Array(nums.length).fill(0);

      continue;
    }

    multi *= num;
 }
```
이렇게 작성할수도 있을 꺼 같습니다!